qiskit_metal/renderers/renderer_gmsh/gmsh_renderer.py

```diff
@@ -156,11 +156,61 @@ def close(self):
         """Public method to close the Gmsh renderer"""
         return self._close_renderer()
 
+    def get_thickness_for_layer_datatype(self,
```

Comment:
@priti-ashvin-shah-ibm I think you have this method in the layerstack code?

Reply:
@ThomasGM4 the layer stack code already has a generalised version of this method (called `get_properties_for_layer_datatype`, which I have called within this local function). I just made this local function because I was extracting thickness a lot of times and felt like make a function to reuse that particular code block multiple times.

Reply:
Ah I see, no worries then. All looks good